kedro/extras/datasets/api/api_dataset.py

```diff
@@ -17,27 +17,26 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
     """``APIDataSet`` loads the data from HTTP(S) APIs.
     It uses the python requests library: https://requests.readthedocs.io/en/latest/
 
-    Example adding a catalog entry with
-    `YAML API
-    <https://kedro.readthedocs.io/en/stable/data/\
-        data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
+    Example using the
```

Comment:
@stichbury Do you think this is an improvement, or less clear? Whatever we decide on, I guess we can replicate consistently for the datasets.